### PR TITLE
refactor: apply ui guidelines to wallet domain modals

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
Add UI guidelines on app & domain components that are used in wallet domain modals.

Note: All modifications are generally minor (spacing mostly), but they affect numerous snapshots as the components affected are commonly used.

* Modified general purpose components spacing and alignment (`Modal`, `TransactionListItem`, `WalletListItem` `Address` component, `Form` wrapper component, `SearchBar`)
* Modified wallet domain component modals (`LedgerWallet`, `ReceiveFunds`, `SignMessage`, `UpdateWalletName`, `VerifyMessage`, `WalletUpdate`)
All models under `domain/wallet` should be displayed

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/app/components/Form/FormLabel.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export function FormLabel(props: FormLabelProps) {
 	return (
 		<label
 			data-testid="FormLabel"
-			className="flex inline-block text-sm FormLabel transition-colors duration-100 text-theme-neutral-dark"
+			className="flex inline-block mb-2 text-sm FormLabel transition-colors duration-100 text-theme-neutral-dark"
```
Comment (luciorubeens) [inline src/app/components/Form/FormLabel.tsx:19]: `FormField` already has `space-y-2`

Comment (goga-m) [reply src/app/components/Form/FormLabel.tsx:19]: Right, for some reason `SignMessage` form labels where stick to the formfield inputs and I patched it with it. Let me check again now.

Comment (goga-m) [reply src/app/components/Form/FormLabel.tsx:19]: Correction: not the `SignMessage` but forms like `VerifyMessage` don't appear to have `space-y-2` and the label sticks to the input. But even those who have `space-y-2` no extra space is added (e.g `EditContact`), they still have `0.5rem` margin in between.

Comment (luciorubeens) [reply src/app/components/Form/FormLabel.tsx:19]: checked the storybook and it looks ok 👍

**Pull request merged**